md/layout-development.md

```diff
@@ -0,0 +1,78 @@
+# Develop a layout
+
+This pages describe how to create a layout and use it in an application.  
+
+## Definition
+
+A layout is structure of the application and page content. It defines menu positioning, a footer, or any common content that could be displayed on every page of the application.
+Each application can have its own layout, or the same layout can be used consistently to all applications.
+
+There is an example of layout in Bonita Portal, that can be found in the Administrator "Resources" section.  
```

Comment:
Maybe we should give its name : Bonita layout (because there is also Default living application layout (deprecated))

Reply:
ok